assembl/static2/js/app/components/common/footer.jsx

```diff
@@ -129,10 +126,4 @@ const mapStateToProps = state => ({
   lang: state.i18n.locale
 });
 
-const withData = graphql(TabsConditionQuery, {
-  props: ({ data }) => ({
-    ...data
-  })
-});
-
-export default compose(connect(mapStateToProps), withData, manageErrorAndLoading({ displayLoader: false }))(Footer);
\ No newline at end of file
+export default connect(mapStateToProps)(Footer);
```

Comment:
Can you please modify `mapStateToProps` above to just return `slug` instead of `debateData`?